tests/queries/0_stateless/03141_fetches_errors_stress.sql

```diff
@@ -14,6 +14,12 @@ system flush logs;
 select event_time_microseconds, logger_name, message from system.text_log where level = 'Error' and message like '%Malformed chunked encoding%' order by 1 format LineAsString;
 
 -- { echoOn }
-select table, errorCodeToName(error), count() from system.part_log where database = currentDatabase() and error > 0 and errorCodeToName(error) not in ('FAULT_INJECTED', 'NO_REPLICA_HAS_PART', 'ATTEMPT_TO_READ_AFTER_EOF') group by 1, 2 order by 1, 2;
+select table, errorCodeToName(error), * from system.part_log where
```

Comment:
leftover from testing?

```suggestion
select table, errorCodeToName(error), from system.part_log where
```